preprocessor/src/main/java/com/azure/autorest/preprocessor/Preprocessor.java

```diff
@@ -225,6 +225,16 @@ public <T> T getValue(Type type, String key) {
         return wrappedPlugin.getValue(type, key);
     }
 
+//    @Override
+//    public <K, V> Map<K, V> getMapValue(Class<K> keyType, Class<V> valueType, String key) {
+//        return wrappedPlugin.getMapValue(keyType, valueType, key);
+//    }
+//
+//    @Override
+//    public <T> List<T> getListValue(Class<T> valueType, String key) {
+//        return wrappedPlugin.getListValue(valueType, key);
+//    }
+
```

Comment:
Why do we need this commented code?

Reply:
I want to add it back in the near future to simplify the calls to get List and Map values but testing has shown that using this results in some code generation errors I haven't figured out yet.